types/v56/types.go

```diff
@@ -1597,7 +1597,7 @@ type OVFItem struct {
 	Reservation     int      `xml:"rasd:Reservation"`
 	ResourceType    int      `xml:"rasd:ResourceType"`
 	VirtualQuantity int64    `xml:"rasd:VirtualQuantity"`
-	Weight          int      `xml:"rasd:Weight"`
+	Weight          int      `xml:"rasd:Weight,omitempty"`
```

Comment:
maybe worth adding comment that Weight is "shares" ?

Reply:
Good suggestion, indeed 🙂

Reply:
Added.